src/CHANGES.txt

```diff
@@ -14,11 +14,14 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Improve finding of Microsoft compiler: add a 'products' wildcard
       in case 2017 Build Tools only is installed as it is considered a separate
       product from the default Visual Studio
+    - Fixed issue running scons-local using a symlink to the scons.py in
```

Comment:
oh is there and issue #  for this, if so please reference?

Reply:
No, the report came in on irc and I don't see a GH issue for it.